zuul.d/jobs.yaml

```
@@ -2139,3 +2139,19 @@
       - OS:ubuntu-xenial
       - Arch:x86_64
       - BuildType:Integration test
+
+# hbase arm build periodic job
+- job:
+    name: hbase-build-arm64
+    parent: init-test
+    description: |
+      Hbase build in openlab arm instance.
+    run: playbooks/hbase-arm-build/run.yaml
+    nodeset: ubuntu-xenial-arm64
+    tags:
+      - Category:BigData
+      - Project:apache/hbase
+      - Application:Hbase@latest-release
```

`latest-release` means released version, `master` means developing branch

```suggestion
      - Application:Hbase@master
```

---

Thanks, Done